populations/models.py

```diff
@@ -208,16 +205,13 @@ def find_submodels(name, obj):
                         df = pd.read_hdf(file_path, key=smdl)
                         # synthesize parameters if not present 
                         #   in the dataframe
-                        df = get_params(df, param_dict.keys(), spinmag)
+                        df = get_params(df, param_dict.keys(), kwargs['spinmag'])
                         label = '/'.join(smdl_list)
                         mdl = KDEModel.from_samples(\
```

Comment:
So, the only time we'll be doing log sampling is with the flows, right? We should be very explicit here that this has to be done consistently in the training of the flows as well as what you feed into the sampler from population-parameter-dict.